### DISCUSSION TITLE
Stories Vivo App

### DISCUSSION THREAD

**Discussion by BiancaMach** [Telefonica/mistica-design#1998] [Proposals]
### Goal

Olá! Na última reunião com Vivo e Mística, falamos um pouco sobre a funcionalidade de Stories no App Vivo que está planejada para Q1 e identificamos possíveis oportunidades de evolução em alguns componentes necessários nessa funcionalidade. Deixo aqui a apresentação da proposta no Figma que também é o arquivo com a funcionalidade detalhada para que possam explorar e entender melhor.

### Context

Criar e definir comportamentos de um componente de Stories escalável para o App

### Impact

A funcionalidade de Stories foi pensada para tornar a interação com os conteúdos do aplicativo mais dinâmica.

### Discussion Questions

No componente naked card, para esse caso, seria necessário que o texto pudesse estar centralizado ou que fosse construído de maneira atômica, se a alteração não for possível.
![Screenshot 2024-12-19 at 22 44 56](https://github.com/user-attachments/assets/83a479bb-93ba-414f-9e47-038903f60ba5)

O progress bar, atualmente, no Figma, só comporta até 6 steps e seria necessário comportar mais que isso para se adequar à proposta
![Screenshot 2024-12-19 at 22 45 19](https://github.com/user-attachments/assets/d1592695-7c14-4add-b417-392850c82cfc)

Fizemos alguns testes também no >> [playroom](https://mistica-web.vercel.app/playroom/#?code=N4Igxg9gJgpiBcIA8UCWA3AOgOwAS4GcAXATwBsYBeYYHffNAgBzIEMT5dMQAjMiMAGtuAGjr0mEAqiKoI2TtwBmqAB4woo8fgDuqKEQAWikAEYADOYCkWvPW4AvALSpssVdxOnb9XIZioAOaGRCYAzJZMHiBidvg8rEKBAE4QAK5uJmSuMKzJTimsaDDYRAAU5rCBIrjJgQkVIuZNTQCUuNY1dQ3NvU0AdABs7RbWrT64AL6TOAB84kgAQhCquExFaNiB1AAck-NxuEgAKv4AtjAAanmorKW46Dd3RJTcF2is3Ae%2B%2BEgAykREoJCOswFRgKZBvttD8kAAFVIpGAEAiLPIAmBMJgaQhETEEagAJihuDAaWSyRKRAxTGoYUmuAA9N8fr8AJLYbLYGAgxJUbg8GBEHQwErcXCsbKBbBsvFnAncMGlGDJL4w1lIDlcnnMPm7BmSoIyuUK8BUlVqw6s34AQUegNVVut%2BGkDnBABZzDMnc6CMkwK8QCEiEwCPBGYzUP0mK5UGdAv1IGdGQB2MKDVSMpRQRk8HMATiUWagecLg1MPAArGAwlAU2nK0pWKweGAeGEwFAdu6wIWU-0AFZMQLcdXW5ljjXHGCqIhhXDvVBpM6kiD8ZLUSDrgj9VzoFUEGDQn3WgAiqDArAIuFgq%2BVAC%2BoBBJ7DGdPZ2EWc6kIytTkv9amqcjkvJgvqAHfmykDYIsaREEQ8gvqyBBnJKZBIT88gIsiBLAGU7SULMuB6G4EA6P0koquU3BMJSKIaAAhNwrTes6rJQfI1AcdgAASuTJEQABKMCBGkbDJKxbG%2BKQ2KBjwyR3JoIAYb4CRJKkGRQMcJCydwBBrvoo4nr4E7GfQmrQbB8GIWZvgoWhKn0FhdG4fhuCEcRrhPuRlECWUNEuYxzGSVJ%2BDcVx0EAML8IewmieJIWhTJ-K8ApbhGaF9BqYIKTpG42m6SA%2BnZEpjmmVJP5-tyEHmb%2BwHVWOP4AkCAE-qcMAXNcyS3KULI-ssqgHD%2BaDoAcOBIJc%2BgwM%2Bdj%2BEEIS7JYkl%2BgG3DBkw4aMkmZzyFArCAsQEAKYEMD9IEEAQIEFCsDGO5JoygREOgTjoFNUhODwaRCEKjIEKwZwsDAb78QQADySgYjAZD9AD7pGaZMQgEYHXIggADaICTegEAAPoAHIwDoIAALoiCAegGIYBDo%2Bm5ikyAGgyEdPH6LA2AIEQyRpEeQA) <<  sobre a funcionalidade

### Do you have a visual proposal?

https://www.figma.com/proto/qsrpbLfB8Jz0tNsDJPtEsz/Stories-no-App?page-id=10244%3A35824&node-id=10264-43781&node-type=frame&viewport=673%2C-1221%2C0.09&t=6TEnEBB09r8OATex-1&scaling=contain&content-scaling=fixed

**Comment by marinitx** [comment]
Hola @BiancaMach, para el Asset + Texto centrado se puede hacer extended de manera sencilla [[(ejemplo)](https://mistica-web.vercel.app/playroom/#?code=N4Igxg9gJgpiBcJgApkEoAEBeAfB4AOgHYYaREDOALhgNoBuAljAO4xQCSVMAthQDQYKMKgDVmbTtz4BdbBgBKMAIZgqAOgCuwgMpVl3ZMAC%2BaANzFipctQwALZUSgAbGONbt5yRk5gAPTFx8K1IhEXdJLl4KVAAHACcYekC8IxDQjHUshKT%2BdNDaH1g-GXgMKnjNGDySUNNzEOMLIhCbGj0IeIBPL2AMbRh4jh5lAHNq-uF4gDllHgmi-wxTbFT8gB4AFQhNMAcAI1cMCCIABUSKCiwUFPtHFzcJdm9fAOMcfNJ1vVUAayFYqoYNcABzvT6hdYAQWcjFGJD8WAI4BgRG48WRH1qGQy6ygjHoEJxQioXVc10I2OJOP2nVg8TKTA8UmihVeMiJ1NIAH4MAADABMsT8QggsKg5XijgogMSaL5nK5ZUFwtF4owABJgJBnJ0KOp9lKnMY%2BTUucTafF6QplPjtGVkQBWAAMAFJkWbzRkAGaMZzoxlPFl8NnFOS85GjKVdChgZSuZAgt1oZEYMqaXy%2BojsT1e0gQQFgRikwPMqIhxYlDARkAAdkd7pAaYwyIAjM63R7FTiWIwoFQ7A6QAA2AAswq7VK9dhgcLsVCHY4nIFzefxMucyi6Q%2B9rj8k7zoQAVtoqIxvV0AMInbhoodgVHog%2BHjDxuFEcsUe%2BPwbPr3GYxuyxQ9oXoAxlHiIRGAAL2BYBHWHYwhHiMBrgGIYRnGJCAHpgPNdZsPxeg8OJAiYXfEicWhWF4QwRFkQfNFfxASjSM2fwqAFbtSG4PwqHI%2BEkRRJiMRAbiyDFTprnE0gmUiaQKFDfxwwk3V4n1XiqB0GByCgCCejKHU9XUTTzkYEZunEwCpxxVjqWAdDZnmayvQI9i%2BIFOyvmwgSiDsgifjAX5KLcnY9mUQ4YDwhoWlqRIqE0eISGQDYlBlE4KAJGAABktx2Kh-IAIQgEVASgfEiFGABNa5W0QryMHWS8IJ2YRnG7YtolOQZTjGOCAGYXPNXsB0KzRnFcKgKA6hTriheJo3Ub14ggHgjAwVxKoHMph2WQRkAAfUEPsAlWDAUps0iOksy7qV%2BGAumuE6hsPdDhj66SQHnKhYi-bDsNiRgwAoTQeHUWI7AgKgIAobC%2B2wgV%2BtTABqDATowVHkQRjtsP6jtkRevNHLmOC%2BXQogSc1YB0dR1sTUJr1KyeqA-AZ6lcO7NA0DZ0gOZsgjir8ELsLS2IMqy3Kuny6LmlMdBrJXEAB14GAKAQWgQAAWQgJhqAgxXxHoCADumVgQBkfgQF7fs7DV%2BBaH64dnX4dsBVHGRjCAA)]

En cuanto al número de steps de la Progress Bar Stepped, siendo un caso puntual, consideramos que lo mejor es hacer un detach del componente y añadir los steps que sean necesarios.

**Comment by yceballost** [comment]
@BiancaMach can we close this discussion?

**Reply by BiancaMach** [reply]
yep!
